src/App.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { generateUniqueID } from 'web-vitals/dist/modules/lib/generateUniqueID';
+
+import TodoList from './TodoList';
+import MenuPopupState from './Menu';
+
+const INITIAL_DATA = [
+  {
+    itemName: 'default item',
+    itemStatus: false,
+    id: generateUniqueID()
+  },
+  {
+    itemName: 'another item',
+    itemStatus: false,
+    id: generateUniqueID()
+  },
+  {
+    itemName: 'and another item',
+    itemStatus: true,
+    id: generateUniqueID()
+  }
+];
+
+const App = () => {
+  // TO ASK should this be in app or todolist
+  // here, todoitems update -> whole list rerender?
+  // in todolist, only updated items rerender?
+  const [todoItems, setTodoItems] = useState(INITIAL_DATA);
```

Comment:
Since you need to call setTodoItems in case of adding a new task, you'll need this state variable here. If it were only used in todolist, then it belongs in todolist.